### ISSUE TITLE
SeaBIOS won't execute (unimplemented opcode that should exist)

### ISSUE THREAD

**Issue by ghost** [ianopolous/JPC#5]
The opcodes `sgdt` and `pause` are unimplemented, so JPC fails when starting SeaBIOS Git or 1.10.2 with an UnimplementedOpcodeException. However, I do see code for `sgdt` at least.

I was able to workaround `pause` by changing it in code to `nop`, but what is wrong with `sgdt`?

**Comment by ianopolous** [comment]
It is probably sgdt in a different operating mode? I'm happy to accept prs if you get it working. The opcode implementations are generated from an xml meta description.

**Comment by ghost** [comment]
Ah, I think I understand the classes now.

the opcodes.vm, pm, and rm classes are the respective operating modes of
the processor. So I need to port the sgdt to whatever mode SeaBIOS is using?

**Comment by ianopolous** [comment]
That's correct. I haven't looked at this in a while so I don't remember in which modes it is implemented. sgdt might be a little involved, but you can check the intel documentation and also the equivalent source code in Bochs and Qemu for comparison after implementing it.
